core/jvm/src/main/scala/zio/metrics/jvm/JvmMetricsSchedule.scala

```diff
@@ -2,8 +2,21 @@ package zio.metrics.jvm
 
 import zio._
 
-final case class JvmMetricsSchedule(value: Schedule[Any, Any, Unit])
+/**
+ * Configuration for the JVM metrics
+ *
+ * @param updateMetrics
+ *   Schedule for periodically updating each JVM metric
+ * @param reloadDynamicMetrics
+ *   Schedule for regenerating the dynamic JVM metrics such as buffer pool
+ *   metrics
+ */
+final case class JvmMetricsSchedule(
+  updateMetrics: Schedule[Any, Any, Unit],
```

Comment:
Let's change the return type of these two schedules to `Any`, e.g.:

```suggestion
  updateMetrics: Schedule[Any, Any, Any],
```

etc.